Assets/MixedRealityToolkit.SDK/Experimental/Features/Dwell/DwellProfile.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
+{
+    [MixedRealityServiceProfile(typeof(DwellProfile))]
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Dwell Profile", fileName = "DwellProfile", order = 100)]
+    [Serializable]
+    public class DwellProfile : ScriptableObject
+    {
+        [Tooltip("Pointer type to use for triggering a dwell interaction")]
+        [SerializeField]
+        private InputSourceType dwellTriggerPointerType = InputSourceType.Head;
+
+        [Tooltip("Delay in seconds until it is determined that the user intends to interact with the target.")]
+        [SerializeField]
+        [Range(0, 2)]
+        private float dwellIntentDelay = 0;
+
+        [Tooltip("Delay in seconds until dwell feedback is started to be shown.")]
```

Comment:
>feedback is started to be shown [](start = 47, length = 31)

Some of these comments are interesting because they suggest feedback/UI, when this portion of the code is designed to be agnostic of that and more about the state transitions - as such, it might be best to keep language here scoped to the data model underlying, instead of the higher level experiences that get built on top of that.

Otherwise, the natural question is "what feedback"? The answer is, there's no feedback in this code, and you have to build it yourself on top of this.